eng/packages/http-client-csharp-mgmt/eng/scripts/Generate.ps1

```diff
@@ -21,7 +21,7 @@ if (-not $LaunchOnly) {
 
         $mgmtTypespecTestProject = Join-Path $testProjectsLocalDir "Mgmt-TypeSpec"
 
-        Invoke (Get-Mgmt-TspCommand "$mgmtTypespecTestProject/main.tsp" $mgmtTypespecTestProject -forceNewProject $ForceNewProject)
+        Invoke (Get-Mgmt-TspCommand "$mgmtTypespecTestProject/main.tsp" $mgmtTypespecTestProject)
```

Comment:
Filed issue: https://github.com/Azure/azure-sdk-for-net/issues/49716

Reply:
It looks like Get-Mgmt-TspCommand doesn't actually have the `forceNewProject` parameter and always generates a new project.